### PR TITLE
add gpg public key. update readme

### PR DESCRIPTION
**Pull request by blacob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ Thank you for using Crusoe Cloud!
...
 ## To install Crusoe Cloud cli with apt:
-- `echo deb https://apt.fury.io/crusoe/ * *" > /etc/apt/sources.list.d/fury.list`
-- `curl https://apt.fury.io/crusoe/gpg.key | sudo apt-key add -`
+- `echo "deb [trusted=yes] https://apt.fury.io/crusoe/ * *" > /etc/apt/sources.list.d/fury.list`
 - `sudo apt update`
-- `sudo apt install crusoe`
\ No newline at end of file
+- `sudo apt install crusoe`
+
+If you are someone is ultra keen on verifying the authenticity of our releases, our public gpg key is located at signing/public.key, and we include a signature of checksums.txt `checksums.txt.sig` as an asset with each of our releases.
```
Comment (lanwall12) [inline README.md]:
```suggestion
If you are someone who is ultra keen on verifying the authenticity of our releases, our public gpg key is located at signing/public.key, and we include a signature of checksums.txt `checksums.txt.sig` as an asset with each of our releases.
```

**Comment by lanwall12** [comment]
I have not used GitHub in so long, this UI looks totally foreign to me.

File: README.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Thank you for using Crusoe Cloud!
...
 ## To install Crusoe Cloud cli with apt:
-- `echo deb https://apt.fury.io/crusoe/ * *" > /etc/apt/sources.list.d/fury.list`
-- `curl https://apt.fury.io/crusoe/gpg.key | sudo apt-key add -`
+- `echo "deb [trusted=yes] https://apt.fury.io/crusoe/ * *" > /etc/apt/sources.list.d/fury.list`
 - `sudo apt update`
-- `sudo apt install crusoe`
\ No newline at end of file
+- `sudo apt install crusoe`
+
+If you are someone who is ultra keen on verifying the authenticity of our releases, our public gpg key is located at signing/public.key, and we include a signature of checksums.txt `checksums.txt.sig` as an asset with each of our releases.
```
Comment (blacob) [inline README.md:15]: cc: @asciimike in case you would like to include such a note on https://docs.crusoecloud.com/quickstart/install-the-cli

**Comment by blacob** [comment]
We might want to make at least 1 approval required for PRs into this repo. 

I don't think I have access, @asciimike or @nitper , do you hold the power?

**Comment by lanwall12** [comment]
@blacob I turned on required PRs and approvals.

**Review by lanwall12 — APPROVED** [review]

**Pull request merged**